get-resource.sh

```diff
@@ -17,24 +17,25 @@ fi
 RHCOS_IMAGE_URL_STRIPPED=`echo $RHCOS_IMAGE_URL | cut -f 1 -d \?`
 if [[ $RHCOS_IMAGE_URL_STRIPPED =~ qcow2.[gx]z$ ]]; then
     RHCOS_IMAGE_FILENAME_RAW=$(basename $RHCOS_IMAGE_URL_STRIPPED)
-    RHCOS_IMAGE_FILENAME_OPENSTACK=${RHCOS_IMAGE_FILENAME_RAW/.[gx]z}
-    IMAGE_FILENAME_EXTENSION=${RHCOS_IMAGE_FILENAME_RAW/$RHCOS_IMAGE_FILENAME_OPENSTACK}
+    RHCOS_IMAGE_FILENAME_QCOW=${RHCOS_IMAGE_FILENAME_RAW/.[gx]z}
+    IMAGE_FILENAME_EXTENSION=${RHCOS_IMAGE_FILENAME_RAW/$RHCOS_IMAGE_FILENAME_QCOW}
     IMAGE_URL=$(dirname $RHCOS_IMAGE_URL_STRIPPED)
 else
     echo "Unexpected image format $RHCOS_IMAGE_URL"
     exit 1
 fi
-RHCOS_IMAGE_FILENAME_COMPRESSED=${RHCOS_IMAGE_FILENAME_OPENSTACK/-openstack/-compressed}
+
+RHCOS_IMAGE_FILENAME_CACHED=cached-${RHCOS_IMAGE_FILENAME_QCOW}
```

Comment:
How about something like 
`RHCOS_IMAGE_FILENAME_CACHED=${RHCOS_IMAGE_FILENAME_OPENSTACK/.qcow2/-compressed.qcow2}`
this would maintain the existing API and remove the need for having "openstack" in the name.

Reply:
Hmm, I think I did the renaming such that it still has the same API.  I tested it locally and it came up with the same result.  I'll be doing full testing today.

Maybe I'm missing something here though?  Is there a flaw in my logic?

Reply:
What I meant was the filename of the compressed qcow2 file has changed but looking at the example I gave it also changes so forget I said anything.

Reply:
Right, I see.. sadly this requires changes here and in the installer.. and I haven't gotten them to work nicely yet.. :/

Reply:
Yes, I think a symlink with the original substitution is what will be required.  If the name has -openstack in it that is.  Inserting a -openstack before .qcow2 is just an insertion where the original expected a substitution so I don't think that will fix the backwards compatibility.